### PR TITLE
Create prometheus postgres exporter chart

### PR DESCRIPTION
**Pull request by gianrubio**

### CODE REVIEW

**Comment by unguiculus** [comment]
Please follow our RBAC best practices.

https://github.com/kubernetes/helm/blob/master/docs/chart_best_practices/rbac.md

**Comment by unguiculus** [comment]
Well, it's not really RBAC, but the part for the service account still applies.

**Comment by gianrubio** [comment]
@unguiculus could you review it again?

File: stable/prometheus-postgres-exporter/templates/serviceaccount.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
```
Comment (unguiculus) [inline stable/prometheus-postgres-exporter/templates/serviceaccount.yaml:2]: Should be conditionally created based on `serviceAccount.create`

Comment (gianrubio) [reply stable/prometheus-postgres-exporter/templates/serviceaccount.yaml:2]: done

Comment (gianrubio) [reply stable/prometheus-postgres-exporter/templates/serviceaccount.yaml:2]: done

**Pull request merged**